cli/src/commands/rebase.rs

```diff
@@ -65,17 +65,14 @@ use crate::ui::Ui;
 /// ```
 ///
 /// With `-b`, the command rebases the whole "branch" containing the specified
-/// revision. A "branch" is the set of commits that includes:
-///
-/// * the specified revision and ancestors that are not also ancestors of the
-///   destination
-/// * all descendants of those commits
+/// revision. A "branch" is the set of revisions that includes all revisions
+/// reachable from the specified revision by via parent or child edges without
+/// visiting an ancestor of the destination.
```

Comment:
I'm not sure if it's reasonable to define "branch" as revisions reachable for both ancestors/descendants directions, but I don't have strong opinion about that.

@arxanas any thoughts?

Reply:
I would call this a "component", or something other than branch.

Reply:
More out-there idea: "wing", like "east wing of a house". It also kinda looks like a wing in a commit graph I'm imagining:

```
|   *
|  /|
| @ *
|/ /
| /
```